tests/in/extra.param.ron

```diff
@@ -1,6 +1,8 @@
 (
 	god_mode: true,
-	spv_version: (1, 0),
+	spv: (
+		version: (1, 2),
```

Comment:
This change doesn't seem to make any substantive difference in the output. Should we just let it default?

Reply:
I thought it's going to be better in the long term if the "extra" test targeted the highest SPIR-V version, since it would need to use some capabilities not available in early versions anyway.